Configuration/FormConfigurationFactory.php

```diff
@@ -130,6 +130,7 @@ private function buildAdminMailConfigurationByTypeAndData(AbstractType $type, $f
         $adminMailConfiguration->setReplyTo($type->getNotifyReplyToMailAddress($formData));
         $adminMailConfiguration->setAddAttachments($type->getNotifySendAttachments($formData));
         $adminMailConfiguration->setTemplate($type->getNotifyMail($formData));
+        $adminMailConfiguration->setPlainTextTemplate($this->mailAdminPlainTextTemplate);
```

Comment:
This templates are not for static forms. And should stay null

Reply:
But it's beeing used by the `Form\Handler::sendMail()` method.